osc-check_dups.py

```diff
@@ -5,8 +5,11 @@
 # Copy this script to ~/.osc-plugins/ or /var/lib/osc-plugins .
 # Then try to run 'osc checker --help' to see the usage.
 
+import osc.conf
+
 def _checker_check_dups(self, project, opts):
-    url = makeurl(opts.apiurl, ['request'], "states=new,review&project=%s&view=collection" % project)
+    user = osc.conf.get_configParser().get(opts.apiurl, "user")
```

Comment:
Rather than reparsing the config it makes sense to just pull the already parsed value out the same way the lock code does.

https://github.com/openSUSE/osc-plugin-factory/blob/4011159df44ce05b3c74a53e29ffe007d1b27a26/osclib/obslock.py#L38